doc/_scripts/redirects.py

```diff
@@ -173,6 +173,7 @@
     ('reference/util/index', 'kernel/util/index'),
     ('samples/drivers/kscan_touch', 'samples/subsys/input/input'),
     ('samples/net/cloud/google_iot_mqtt', 'samples/net/cloud'),
+    ('samples/subsys/display/lvgl', 'samples/modules/lvgl/helloworld'),
```

Comment:
actually, the syntax is probably more confusing than it needs to be but it should be:

```suggestion
    ('samples/subsys/display/lvgl/README', 'samples/modules/lvgl/helloworld/README'),
```